scripts/web3_scan_utils.py

```
@@ -0,0 +1,93 @@
+
+def checksum_address(address: str) -> ChecksumAddress:
+    from web3 import Web3
+
+    return Web3.to_checksum_address(address)
+
+
+def execute_read_function(
+    contract_address: str,
+    abi: str,
+    function_name: str,
+    function_parameters: List[str] = [],
```

Ruff (0.7.0)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)